src/components/shared/Login.tsx

```diff
@@ -0,0 +1,81 @@
+import React from "react";
```

Comment:
You can move the Login component from shared to auth folder.

Reply:
Done